data/hai/hai reveal 4 middle.txt

```diff
@@ -0,0 +1,462 @@
+			choice
+				`	(Enter the building.)`
+			`	You're ushered to an elevator and given a room number: 3017A19, the 301st floor, section 7A, room 19. It's amazing how large Earth's megabuildings have become. Some planets have lower population than one of these buildings.`
+			`	Giti is even more excited in-person than she was by holocommunicator. She's already researched the Syndicate-like symbol, but needed more detail about the demonic one. You also show her the information about the microtransceivers.`
+			`	"This cat symbol was used by a clandestine organization that worked within the Syndicate over a half-century ago. They called themselves 'Base Principles.' They were anarcho-syndicalists infiltrating the Syndicate from within, supposedly advancing their positions through research findings gleaned from captured alien equipment, but nobody knows where they got it. The organization was eventually torn in two by a collection of regular anarchists and other unsavory characters looking to profit off the backdoor access to the Syndicate; this group kidnapped researchers from other institutes to expand their capabilities."`
+			`	"These others didn't cover their tracks well enough, the trail led back to the syndicalists that started it all in the first place, and Syndicate Internal Affairs put a stop to the organization with extreme prejudice. Most of the anarchist leaders were killed in battle or executed after capture, the internal players simply... disappeared."`
+			`	"Officially the Syndicate maintains that they wiped Base Principles out to the last individual, but evidence never did prove it had been conclusively terminated."`
+			`	"For the demonic symbol, we'll have to go to another library in room 4196A21."`
+			choice
+				`	(Take the elevator to the 419th floor.)`
+			`	When you arrive, Giti tells you, "This is a religious materials archive. Give me a minute here..." Her voice trails off as she starts typing on a terminal. She walks back and forth between several bookshelves skimming books for a few minutes.`
+			`	"I found it," she says. "You found a modified version of the Symbol of Baphomet, the official symbol of an older incarnation of the Church of Satan. They were one of the first organizations to fund large-scale settlements on Freedom, though the symbol was ultimately co-opted by miscreants. The letters around the pentagram have been replaced with galactic coordinates using an archaic notation." She smiles at you, "I wrote my Ph.D. thesis about that notation! Now I finally need that information for something other than getting a degree!"`
+			`	"The circle around the symbol looks almost like a wormhole. A red wormhole. I heard about one in Terminus near The Deep. Apparently nobody is able to enter one without their ship being torn to pieces. This symbol suggests there's another one in the middle of Syndicate space."`
```

Comment:
Using an Ember Waste wormhole hole raises some unanswered questions:
* All Ember Waste wormholes need to be stabilised with keystones, and all other non-Ember Waste wormholes don't. What makes this one different?
* If you have a jump drive and a keystone (both of which are apparently required) and are willing to explore strange region of space, it's kind of hard to not run into the Remnant.
* Even if you don't run into the Remnant, surely they have access to jump drives (perhaps only have a handful, like the Syndicate) and would probably have found this area of space at some point. If nothing else, an Ember Waste wormhole outside the Ember Waste would surely be something they'd want to take a closer look at.
* If this whole thing predates contact with the Hai (as implied by Teeneep earlier) and they don't have contact with the Remnant, where would they have acquired a keystone?

I think it would make more sense for this to be a normal wormhole, or require some new kind of key that would be more easily available in human space. Perhaps a "philosopher's stone"?

Reply:
You have 4 questions here:
1. Just because wormholes in the ember waste are unstable, does not mean that unstable wormholes may only exist in the ember waste.
2. & 3. I would like to use a system hazard to apply cloak disruption in one of the systems beyond it, thus acting as a stern discouragement for the Remnant to have been around here because they don't like meeting people.
4. The fact that the organisation may pre-date contact with the Hai does not necessarily mean that their current base of operations and general MO predates it. Plus, the Hai previously occupied this part of space. who's to say that there wasn't the odd keystone unearthed from times past that was accidentally left behind. ;) (This is something I would like to expand upon in a side-quest later, but it's not required for the main PR.)

Reply:
All good points. Perhaps putting the Ember Waste hazard in the Devil-Run system to make it more of a disconnected section of the Waste (it's certainly close enough for that to work) would avoid questions and also make the area just that little bit less hospitable?

Reply:
This is waiting on #3599